models/Series.js

```diff
@@ -1,7 +1,11 @@
 import mongoose from 'mongoose';
-import makeCredential from 'common/makeCredential';
-const mongodb = makeCredential('mongodb');
-const db = mongoose.createConnection(mongodb);
+import mockgoose from 'mockgoose';
+if (process.env.NODE_ENV === 'test') {
+  mockgoose(mongoose);
+}
+const mongodbCredential = process.env.KINDLIZED_MONGODB;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).